main.tf

```diff
@@ -133,3 +133,14 @@ resource "aws_apigatewayv2_integration" "this" {
     ignore_changes = [passthrough_behavior]
   }
 }
+
+# VPC Link (Private API)
+resource "aws_apigatewayv2_vpc_link" "this" {
```

Comment:
This resource became available in [v2.59.0](https://github.com/terraform-providers/terraform-provider-aws/blob/release/2.x/CHANGELOG.md#2590-april-23-2020). Please update `versions.tf`.